fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/ByteArrayContinuation.java

```diff
@@ -34,10 +35,13 @@
 public class ByteArrayContinuation implements RecordCursorContinuation {
     @Nonnull
     private final byte[] bytes;
+    @Nonnull
+    private ByteString byteString;
 
     @SpotBugsSuppressWarnings("EI2")
     private ByteArrayContinuation(@Nonnull final byte[] bytes) {
         this.bytes = bytes;
+        this.byteString = ByteString.copyFrom(bytes);
```

Comment:
Should this be a lazy copy? I believe the consequence of this as written is it will always proactively make a copy and put it in a `ByteString`, even if no one calls `.toByteString()`.

Reply:
Good point, fixed it. Thanks! @alecgrieser